EnglishTranslation-master/ITEM.tsv

```diff
@@ -6312,7 +6312,7 @@ ITEM_20150918_006311	A document recorded in the time of the Great King Zachariel
 ITEM_20150918_006312	Warning notices from hundreds of years ago. Read what's inside.
 ITEM_20150918_006313	A carelessly crumpled message. It seems barely readable.
 ITEM_20150918_006314	Varkis' completed research reports. Contains information related to the Royal Mausoleum. Readable.
-ITEM_20150918_006315	A record written by Hones and Liars's teacher. Say that it contains very important information.
+ITEM_20150918_006315	A record written by Hones and Liars teacher. Say that it contains very important information.
```

Comment:
A record written by the teacher of Hones and Raius. It seems to contain very important information.